### ISSUE TITLE
"vehicles" should be "wheelbarrow"

### ISSUE THREAD

**Issue by paulon6132** [rdrg109/npcr-subtitles#896]
https://github.com/rdrg109/subtitles-npcr/blob/602f3099ee1a51910b0b2b6b31e261e7bc4df630/sentences/42-text-2.yaml#L216-L220

In the video, no vehicles appear, so probably "vehicles" is not the correct word here. 

[output.webm](https://github.com/rdrg109/subtitles-npcr/assets/74389646/4d91c2fb-2a8d-41b0-920b-20bd4c5e9185)

I think the correct translation should be "whellbarrow".

Definition of "vehicles" in [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/vehicle?q=vehicles):

> a machine, usually with wheels and an engine, used for transporting people or goods, especially on land: 

Definition of "wheelbarrow" in [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/wheelbarrow):

> a large, open container for moving things in with a wheel at the front and two handles at the back, used especially in the garden

**Comment by rdrg109** [comment]
I agree with you that it shouldn't be "vehicles". Another reason is that the video shows wheelbarrows.

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/15db3c85-46a8-40cb-a811-16ccaa77bfee)

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/2e908849-51fd-4d01-a49d-e2d651fb7175)

However, I'm not sure what's the correct translation. [mdbg.net](https://www.mdbg.net/chinese/dictionary?page=worddict&wdrst=0&wdqb=推车) also shows "cart" and "trolley" as translations for "推车".
